digits/extensions/data/interface.py

```diff
@@ -109,3 +109,27 @@ def itemize_entries(self, stage):
         this function in no particular order
         """
         raise NotImplementedError
+
+    @staticmethod
+    def can_explore():
```

Comment:
I didn't because get_data is expensive to compute and the data would be thrown away in that context.